Classes/TypesetKit.h

```diff
@@ -10,11 +10,29 @@
 
 @interface TypesetKit : NSObject
 
+typedef enum {
```

Comment:
Added this enum which would be good to use on underline, but I'm using on strikethrough. That way the user can understand the settings for the strikethrough.